### DISCUSSION TITLE
CoPilot Not Working

### DISCUSSION THREAD

**Discussion by ghost** [community/community#108022] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

On VS Code, CoPilot is enabled and says its running, but solutions are not synthesized; it now giving suggestions.

**Comment by yohikofox** [comment]
I got same issue, copilot chat is still working but nothing from suggestions.

**Reply by yohikofox** [reply]
here a few information about one of my requests :

2024-02-12 12:36:21.812 [info] [default] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 10016 ms
2024-02-12 12:36:21.814 [info] [streamChoices] solution 0 returned. finish reason: [Iteration Done]
2024-02-12 12:36:21.814 [info] [streamChoices] request done: headerRequestId: [46bc4e26-7135-4968-8889-7d496559d2a4] model deployment ID: [w1bad1925db0f]

**Comment by robinmalik** [comment]
Dupe: https://github.com/orgs/community/discussions/108008

**Comment by Akash1134** [comment]
👀 Update - We are starting to see recovery based on the signals that the team have been monitoring, following mitigation steps being taken. When confident that recovery is complete, we will resolve this incident.

[Source](https://www.githubstatus.com/)

**Comment by Akash1134** [comment, marked as answer]
Hi everyone,

This issue has now been fixed. If you continue to see issues, please let us know.

Source: [GitHub Status](https://www.githubstatus.com/)

You may self troubleshoot some of the [common issues with Copilot](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-common-issues-with-github-copilot#github-copilot-not-working-in-some-files) using our Docs.

Thanks!

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/108008
